sources/packages/web/src/components/common/ScholasticStandingForm.vue

```diff
@@ -48,13 +48,13 @@ export default defineComponent({
       default: false,
     },
   },
-  setup(props: any, context: SetupContext) {
+  setup(props, context: SetupContext) {
     const data = ref(
       {} as ScholasticStanding | ScholasticStandingBeforeSubmission,
     );
 
     const setReadOnlyData = () => {
-      data.value = { ...props.initialData, readOnly: props.readOnly };
+      data.value = { ...props.initialData, readonly: props.readOnly };
```

Comment:
Nice catch but I believe that the typescript property is wrong in this case when we look into the form definition. Does it make sense? I believe that at runtime it was probably working and this change would break it.
![image](https://user-images.githubusercontent.com/61259237/235813207-dfeb501f-ead4-4893-b5c8-d99745503ef7.png)